packages/shopify-app-remix/src/server/clients/storefront/types.ts

```diff
@@ -13,10 +13,10 @@ export interface StorefrontContext {
    * <description>Use `storefront.graphql` to make query / mutation requests.</description>
    * ```ts
    * import { ActionFunctionArgs } from "@remix-run/node";
-   * import { authenticate } from "../shopify.server";
+   * import { unauthenticated } from "../shopify.server";
```

Comment:
I wonder if we should use `appProxy` as our "official" example here - we want to encourage folks to rely on the authenticated methods as much as we can.

Reply:
Thanks, @paulomarg! I just added the `appProxy` example. Would you mind taking another look to confirm all looks correct?